examples/NavigationPlayground/js/SimpleTabs.js

```diff
@@ -11,21 +11,20 @@ import {
 } from 'react-native';
 import {
   TabNavigator,
+  withNavigation,
 } from 'react-navigation';
 import Ionicons from 'react-native-vector-icons/Ionicons';
 import SampleText from './SampleText';
 
+const ButtonWithNavigation = withNavigation(({ navigation, to, ...rest}) => {
```

Comment:
@satya164 @ericvicenti You are right. I reverted changes in this example and created a new one.